charts/plane-ce/templates/workloads/api.deployment.yaml

```
@@ -76,4 +76,44 @@ spec:
+    - type: Resource
+      resource:
+        name: memory
+    {{- if .Capabilities.APIVersions.Has "autoscaling/v2"}}
+        target:
+          type: Utilization
+          averageUtilization: {{ .Values.autoscaling.targetMemoryUtilizationPercentage }}
+    {{- else }}
+        targetAverageUtilization: {{ .Values.autoscaling.targetMemoryUtilizationPercentage }}
+    {{- end }}
+    {{- end }}
 ---
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)